tests/utils/test_requirements_utils.py

```diff
@@ -192,6 +193,8 @@ def test_prune_packages():
 
 
 def test_capture_imported_modules():
+    from mlflow.utils._capture_modules import _CaptureImportedModules
```

Comment:
Import `_CaptureImportedModules` here to avoid encountering `ImportError` for numpy (`_capture_modules` requires `mlflow.pyfunc` which requires `numpy`) when accessing `test_infer_requirements_excludes_mlflow` in the skinny client test.